tests/unit/src/main/scala/tests/BaseTypeDefinitionLspSuite.scala

```diff
@@ -0,0 +1,111 @@
+package tests
+
+import scala.meta.internal.metals.TextEdits
+import org.eclipse.{lsp4j => l}
+import scala.concurrent.Future
+import scala.meta.inputs.Input
+import scala.meta.inputs.Position
+import scala.meta.internal.mtags.MtagsEnrichments._
+
+class BaseTypeDefinitionLspSuite extends BaseLspSuite("typeDefinition") {
+  def check(
+      name: String
+  )(query: String, expectedLocs: List[String] = Nil): Unit = {
```

Comment:
Is `expectedLocs` used still?

Reply:
`ext-library` uses it. It makes it possible to specify exact position in another file.